src/Dependencies/xUnit.net/project.json

```diff
@@ -0,0 +1,10 @@
+{
+  "supports": {},
+  "dependencies": {
+    "xunit": "2.1.0",
+    "xunit.runner.console": "2.1.0"
```

Comment:
This is the new dependencies projects - they unify dependencies, so that there's a single version of shared binaries across the tree, and a single place to change versions.